### ISSUE TITLE
Release 4.10.0 - Alpha 2 - Wazuh UI regression testing

### ISSUE THREAD

**Issue by asteriscos** [wazuh/wazuh#26431]
The following issue aims to run manual tests for the current release candidate, report the results, and open new issues for any encountered errors.

## Wazuh UI tests information
|                                        |     |
|----------------------------------------|-----|
| **Main release candidate issue**       |  https://github.com/wazuh/wazuh/issues/26425  |
| **Version**                            |   4.10.0  |
| **Release candidate #**                |  Alpha2  |
| **Tag**                                |  [v4.10.0-alpha2](https://github.com/wazuh/wazuh-dashboard-plugins/releases/tag/v4.10.0-alpha2)   |
| **Previous UI regression tests issue** |  https://github.com/wazuh/wazuh/issues/26342 |

## Test report procedure

**TL;DR**
   1. The specified tests will be executed in every platform and version mentioned in this issue.
   1. Include evidence of each test performed.
   1. Report any problem or bug. Open a new issue for each of them and link them here.
   1. Justify skipped tests.

All test results must have one the following statuses: 
|                 |                                                                      |
|-----------------|----------------------------------------------------------------------|
| :white_circle:  | Doesn't apply |
| :black_circle:  | The test hasn't started yet.                                         |
| :green_circle:  | All checks passed.                                                   |
| :red_circle:    | There is at least one failed check.                                  |
| :yellow_circle: | There is at least one expected fail or skipped test and no failures. |

Any failing test must be properly addressed with a new issue, detailing the error and the possible cause.
It must be included in the `Problems` section of the current release candidate issue.

Any expected fail or skipped test must be justified with a reason. 
All auditors must validate the justification for an expected fail or skipped test.

An extended report of the test results must be attached as a zip, txt or images. 
This report can be used by the auditors to dig deeper into any possible failures and details.

## Test :red_circle: 

| Test                                                                 | Chrome         | Firefox        | Safari         |
|---------------------------------------------------------------------|----------------|----------------|----------------|
| Verify the app package installs and operates as expected | :green_circle:    | :green_circle:  | 🟢   |
| [Fix the filter are displayed cropped on screens of 575px to 767px #7047](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7047) | :green_circle:   | :green_circle:   | 🟢  |
| [Re-design the Malware detection dashboard with and without a pinned agent #6964](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6964) | :green_circle:   | :green_circle:   | 🟢  |
| [Add visualization of vulnerabilities evaluation status count #6968](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6968) | :green_circle:   | :green_circle:   | 🟢  |
| [Fix data grid pagination with relative index per page #7044](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7044) | :green_circle:   | :green_circle:   | 🟢  |
| [Rename vulnerability.under_evaluation field #7046](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7046) | :green_circle:   | :green_circle:   | 🟢  |
| [Change the Endpoint security > Malware detection module filters to "rule.groups is one of rootcheck,virustotal,yara" #6963](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6963) | :green_circle:   | :green_circle:   | 🟢  |
| [Change the registration id of the setting application #6938](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6938) | :green_circle:   | :green_circle:   | 🟢  |
| [Changed Mitre overview description #7032](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7032) | :green_circle:   | :green_circle:   | 🟢  |
| [Change agents summary no results #7041](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7041) | :red_circle:   | :red_circle:   | 🔴   |
| [Update malware feature description #7036](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7036) | :green_circle:   | :green_circle:   | 🟢  |
| [Increase the font size of the kpi subtitles and the features descriptions #7033](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7033) | :green_circle:   | :green_circle:   | 🟢  |
| [Containers have different margins #7034](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7034) | :green_circle:   | :green_circle:   | 🟢  |
| [Set a fixed width to the default columns on each selected field #7059](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7059) | :green_circle:   | :green_circle:   | 🟢    |
| [Fix readonly user cannot read statistics #7001](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001) | :green_circle:   | :green_circle:   | 🟢  |
| [Fixed no agent alert spawn with selected agent in agent welcome #7029](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7029) | :green_circle:   | :green_circle:   | 🟢  |
| [Filter deprecated actions #7042](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7042) | :green_circle:   | :green_circle:   | 🟢  |
| [Fix export formatted csv data with special characters from tables #7048](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7048) | :green_circle:   | :green_circle:   | 🟢  |
| [Fix column reordering feature #7072](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7072) | :green_circle:   | :green_circle:   | 🟢  |
| [Avoid filter hiding when adding too many of them #7077](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7077) | :green_circle:   | :green_circle:   | 🟢  |
| [Removed agent RBAC filters from dashboard queries #6945](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6945) | :green_circle:   | :green_circle:   | 🟢  |
| [Remove virustotal feature and dashboards in favor of malware dashboard #7038](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038) | :green_circle:   | :green_circle:   | 🟢  |
| [Remove setting new home #282](https://github.com/wazuh/wazuh-dashboard/pull/282) | :green_circle:   | :green_circle:   | 🟢  |

## Test plan

1. Verify the app package installs and operates as expected
2. [Fix the filter are displayed cropped on screens of 575px to 767px #7047](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7047)
3. [Re-design the Malware detection dashboard with and without a pinned agent #6964](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6964)
4. [Add visualization of vulnerabilities evaluation status count #6968](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6968)
5. [Fix data grid pagination with relative index per page #7044](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7044)
6. [Rename vulnerability.under_evaluation field #7046](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7046)
7. [Change the Endpoint security > Malware detection module filters to "rule.groups is one of rootcheck,virustotal,yara" #6963](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6963)
8. [Change the registration id of the setting application #6938](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6938)
9. [Changed Mitre overview description #7032](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7032)
10. [Change agents summary no results #7041](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7041)
11. [Update malware feature description #7036](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7036)
12. [Increase the font size of the kpi subtitles and the features descriptions #7033](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7033)
13. [Containers have different margins #7034](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7034)
14. [Set a fixed width to the default columns on each selected field #7059](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7059)
15. [Fix readonly user cannot read statistics #7001](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001)
16. [Fixed no agent alert spawn with selected agent in agent welcome #7029](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7029)
17. [Filter deprecated actions #7042](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7042)
18. [Fix export formatted csv data with special characters from tables #7048](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7048)
19. [Fix column reordering feature #7072](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7072)
20. [Avoid filter hiding when adding too many of them #7077](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7077)
21. [Removed agent RBAC filters from dashboard queries #6945](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6945)
22. [Remove virustotal feature and dashboards in favor of malware dashboard #7038](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038)
23. [Remove setting new home #282](https://github.com/wazuh/wazuh-dashboard/pull/282)

## Conclusions 🔴 

An issue was created to add a loading to the Agents summary display in the overview. 
All other tests worked as expected

### Issue created
- https://github.com/wazuh/wazuh-dashboard-plugins/issues/7114

## Auditors validation
The definition of done for this one is the validation of the conclusions and the test results from all auditors.

All checks from below must be accepted in order to close this issue.

- [x] @asteriscos

**Comment by yenienserrano** [comment]
Quickstart

```console
curl -sO https://packages-dev.wazuh.com/4.10/wazuh-install.sh && sudo bash ./wazuh-install.sh -a -d pre-release
21/10/2024 13:34:47 INFO: Starting Wazuh installation assistant. Wazuh version: 4.10.0
21/10/2024 13:34:47 INFO: Verbose logging redirected to /var/log/wazuh-install.log
21/10/2024 13:34:47 INFO: The recommended systems are: Red Hat Enterprise Linux 7, 8, 9; CentOS 7, 8; Amazon Linux 2; Ubuntu 16.04, 18.04, 20.04, 22.04.
21/10/2024 13:34:47 WARNING: The current system does not match with the list of recommended systems. The installation may not work properly.
21/10/2024 13:34:47 INFO: Verifying that your system meets the recommended minimum hardware requirements.
21/10/2024 13:34:47 INFO: Wazuh web interface port will be 443.
21/10/2024 13:34:47 INFO: --- Dependencies ---
21/10/2024 13:34:47 INFO: Installing lsof.
21/10/2024 13:34:49 INFO: Wazuh development repository added.
21/10/2024 13:34:49 INFO: --- Configuration files ---
21/10/2024 13:34:49 INFO: Generating configuration files.
21/10/2024 13:34:49 INFO: Generating the root certificate.
21/10/2024 13:34:49 INFO: Generating Admin certificates.
21/10/2024 13:34:49 INFO: Generating Wazuh indexer certificates.
21/10/2024 13:34:49 INFO: Generating Filebeat certificates.
21/10/2024 13:34:49 INFO: Generating Wazuh dashboard certificates.
21/10/2024 13:34:50 INFO: Created wazuh-install-files.tar. It contains the Wazuh cluster key, certificates, and passwords necessary for installation.
21/10/2024 13:34:50 INFO: --- Wazuh indexer ---
21/10/2024 13:34:50 INFO: Starting Wazuh indexer installation.
21/10/2024 13:35:38 INFO: Wazuh indexer installation finished.
21/10/2024 13:35:38 INFO: Wazuh indexer post-install configuration finished.
21/10/2024 13:35:38 INFO: Starting service wazuh-indexer.
21/10/2024 13:35:49 INFO: wazuh-indexer service started.
21/10/2024 13:35:49 INFO: Initializing Wazuh indexer cluster security settings.
21/10/2024 13:35:53 INFO: Wazuh indexer cluster security configuration initialized.
21/10/2024 13:35:53 INFO: Wazuh indexer cluster initialized.
21/10/2024 13:35:53 INFO: --- Wazuh server ---
21/10/2024 13:35:53 INFO: Starting the Wazuh manager installation.
21/10/2024 13:36:36 INFO: Wazuh manager installation finished.
21/10/2024 13:36:36 INFO: Wazuh manager vulnerability detection configuration finished.
21/10/2024 13:36:36 INFO: Starting service wazuh-manager.
21/10/2024 13:36:48 INFO: wazuh-manager service started.
21/10/2024 13:36:48 INFO: Checking Wazuh API connection
21/10/2024 13:36:48 INFO: Wazuh API is ready to receive requests.
21/10/2024 13:36:48 INFO: Wazuh API connection successful
21/10/2024 13:36:48 INFO: Starting Filebeat installation.
21/10/2024 13:37:27 INFO: Filebeat installation finished.
21/10/2024 13:37:28 INFO: Filebeat post-install configuration finished.
21/10/2024 13:37:28 INFO: Starting service filebeat.
21/10/2024 13:37:28 INFO: filebeat service started.
21/10/2024 13:37:28 INFO: Checking Filebeat connection
21/10/2024 13:37:29 INFO: Filebeat connection successful
21/10/2024 13:37:29 INFO: --- Wazuh dashboard ---
21/10/2024 13:37:29 INFO: Starting Wazuh dashboard installation.
21/10/2024 13:38:52 INFO: Wazuh dashboard installation finished.
21/10/2024 13:38:52 INFO: Wazuh dashboard post-install configuration finished.
21/10/2024 13:38:52 INFO: Starting service wazuh-dashboard.
21/10/2024 13:38:52 INFO: wazuh-dashboard service started.
21/10/2024 13:38:52 INFO: Updating the internal users.
21/10/2024 13:38:55 INFO: A backup of the internal users has been saved in the /etc/wazuh-indexer/internalusers-backup folder.
21/10/2024 13:39:04 INFO: The filebeat.yml file has been updated to use the Filebeat Keystore username and password.
21/10/2024 13:39:31 INFO: Initializing Wazuh dashboard web application.
21/10/2024 13:39:31 INFO: Wazuh dashboard web application not yet initialized. Waiting...
21/10/2024 13:39:47 INFO: Wazuh dashboard web application not yet initialized. Waiting...
21/10/2024 13:40:02 INFO: Wazuh dashboard web application initialized.
21/10/2024 13:40:02 INFO: --- Summary ---
21/10/2024 13:40:02 INFO: You can access the web interface https://<wazuh-dashboard-ip>:443
    User: admin
    Password: 7xZ?EPIpw71+vJarWPEzpZ2X?cjcJ??L
21/10/2024 13:40:02 INFO: --- Dependencies ---
21/10/2024 13:40:02 INFO: Removing lsof.
21/10/2024 13:40:25 INFO: Installation finished.
```

Agent

```console
curl -o wazuh-agent-4.10.0-1.x86_64.rpm https://packages-dev.wazuh.com/pre-release/yum/wazuh-agent-4.10.0-1.x86_64.rpm && sudo WAZUH_MANAGER='ec2-44-220-49-172.compute-1.amazonaws.com' WAZUH_AGENT_GROUP='default' WAZUH_AGENT_NAME='Agent-Test' rpm -ihv wazuh-agent-4.10.0-1.x86_64.rpm
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 9145k  100 9145k    0     0  26.5M      0 --:--:-- --:--:-- --:--:-- 26.5M
warning: wazuh-agent-4.10.0-1.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID 29111145: NOKEY
Verifying...                          ################################# [100%]
Preparing...                          ################################# [100%]
Updating / installing...
   1:wazuh-agent-4.10.0-1             ################################# [100%]
```

**Comment by yenienserrano** [comment]
## Test :black_circle: 

| Test                                                                 | Chrome         | Firefox        | Safari         |
|---------------------------------------------------------------------|----------------|----------------|----------------|
| Verify the app package installs and operates as expected | :black_circle:    | :black_circle:  | :black_circle:  |
| [Fix the filter are displayed cropped on screens of 575px to 767px #7047](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7047) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Re-design the Malware detection dashboard with and without a pinned agent #6964](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6964) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Add visualization of vulnerabilities evaluation status count #6968](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6968) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Fix data grid pagination with relative index per page #7044](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7044) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Rename vulnerability.under_evaluation field #7046](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7046) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Change the Endpoint security > Malware detection module filters to "rule.groups is one of rootcheck,virustotal,yara" #6963](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6963) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Change the registration id of the setting application #6938](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6938) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Changed Mitre overview description #7032](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7032) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Change agents summary no results #7041](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7041) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Update malware feature description #7036](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7036) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Increase the font size of the kpi subtitles and the features descriptions #7033](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7033) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Containers have different margins #7034](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7034) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Set a fixed width to the default columns on each selected field #7059](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7059) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Fix readonly user cannot read statistics #7001](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Fixed no agent alert spawn with selected agent in agent welcome #7029](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7029) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Filter deprecated actions #7042](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7042) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Fix export formatted csv data with special characters from tables #7048](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7048) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Fix column reordering feature #7072](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7072) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Avoid filter hiding when adding too many of them #7077](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7077) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Removed agent RBAC filters from dashboard queries #6945](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6945) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Remove virustotal feature and dashboards in favor of malware dashboard #7038](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038) | :black_circle:   | :black_circle:   | :black_circle:  |
| [Remove setting new home #282](https://github.com/wazuh/wazuh-dashboard/pull/282) | :black_circle:   | :black_circle:   | :black_circle:  |


## Test plan

1. Verify the app package installs and operates as expected
2. [Fix the filter are displayed cropped on screens of 575px to 767px #7047](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7047)
3. [Re-design the Malware detection dashboard with and without a pinned agent #6964](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6964)
4. [Add visualization of vulnerabilities evaluation status count #6968](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6968)
5. [Fix data grid pagination with relative index per page #7044](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7044)
6. [Rename vulnerability.under_evaluation field #7046](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7046)
7. [Change the Endpoint security > Malware detection module filters to "rule.groups is one of rootcheck,virustotal,yara" #6963](https://github.com/wazuh/wazuh-dashboard-plugins/issues/6963)
8. [Change the registration id of the setting application #6938](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6938)
9. [Changed Mitre overview description #7032](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7032)
10. [Change agents summary no results #7041](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7041)
11. [Update malware feature description #7036](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7036)
12. [Increase the font size of the kpi subtitles and the features descriptions #7033](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7033)
13. [Containers have different margins #7034](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7034)
14. [Set a fixed width to the default columns on each selected field #7059](https://github.com/wazuh/wazuh-dashboard-plugins/issues/7059)
15. [Fix readonly user cannot read statistics #7001](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001)
16. [Fixed no agent alert spawn with selected agent in agent welcome #7029](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7029)
17. [Filter deprecated actions #7042](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7042)
18. [Fix export formatted csv data with special characters from tables #7048](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7048)
19. [Fix column reordering feature #7072](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7072)
20. [Avoid filter hiding when adding too many of them #7077](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7077)
21. [Removed agent RBAC filters from dashboard queries #6945](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6945)
22. [Remove virustotal feature and dashboards in favor of malware dashboard #7038](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038)
23. [Remove setting new home #282](https://github.com/wazuh/wazuh-dashboard/pull/282)

**Comment by asteriscos** [comment]
Test looks good to me :heavy_check_mark:
